diagnostics/collect-wsl-logs.ps1

```diff
@@ -130,6 +130,100 @@ if ($Dump)
     }
 }
 
+# Collect networking state relevant for WSL
+$wslNetworkLog = "$folder/wsl_network_snapshot.log"
+$stdOutLog = "$folder/wsl_network_stdout.log"
+$stdErrLog = "$folder/wsl_network_stderr.log"
+
+try
+{
+    Write-Output "WSL Net Snapshot" | Out-File -FilePath $wslNetworkLog
```

Comment:
Instead of merging everything in the same file, could we use the same model as the rest of the script and have each command write its output in a single file ? This would allow us to get rid of the: '-----------------------------------------------------------------------' prints and would remove the need to try / catch this section as long as we pass `-ErrorAction Ignore` to all commands

Reply:
updated to output to multiple files.
will keep the try/catch based on offline discussion